src/components/tvproviders/schedulesdirect.js

```diff
@@ -264,18 +264,20 @@ define(["jQuery", "loading", "emby-checkbox", "listViewStyle", "emby-input", "em
         self.init = function () {
             options = options || {};
 
-            // Show cancel button by default
-            if (options.showCancelButton !== false) {
-                page.querySelector(".btnCancel").classList.remove("hide");
-            } else {
+            // Only hide the button if explicitly set to false; default to showing if undefined or null
+            // FIXME: rename this option to clarify logic
+            if (options.showCancelButton === false) {
                 page.querySelector(".btnCancel").classList.add("hide");
+            } else {
+                page.querySelector(".btnCancel").classList.remove("hide");
```

Comment:
What about this:

```
const hideCancelButton = showCancelButton === false;
page.querySelector(".btnCancel").classList.toggle("hide", hideCancelButton);
`````

This seems pretty clear to me and will also make the transition to rename the option simpler later on